mealy/preprocessing.py

```diff
@@ -250,55 +209,120 @@ def _inverse_single_step(single_step, step_output):
             original_feature_ids, preprocessed_feature_ids = self._get_feature_ids_related_to_transformer(transformer_feature_names)
             output_of_transformer = preprocessed_x[:, preprocessed_feature_ids]
 
-            is_cat = np.vectorize(self.is_categorical)
-            any_numeric = np.any(~is_cat(original_feature_ids))
+            any_numeric = np.any(~self.is_categorical(original_feature_ids))
```

Comment:
We need to vectorize the `self.is_categorical` function otherwise now it returns an error when receiving an array as argument (`TypeError : only integer scalar arrays can be converted to a scalar index`)

This line fixed the issue (tested on my instance). Or you can turn it into a one liner if you think it is better. 

```suggestion
           is_categorical_vectorized = np.vectorize(self.is_categorical)
            any_numeric = np.any(~is_categorical_vectorized(original_feature_ids))
```

Reply:
Addressed in 8231d8b